deploy-metallb/deploy.sh

```diff
@@ -298,6 +294,10 @@ if ! ./verify.sh; then
         verify_remote_resource ${edgecluster} "default" "crd" "metallbs.metallb.io" "."
         echo
 
+        # "Patch bz https://bugzilla.redhat.com/show_bug.cgi?id=2106840"
+        echo ">> Patching the MetalLB operator"
+        ${SSH_COMMAND} -i ${RSA_KEY_FILE} core@${EDGE_NODE_IP}  "oc adm policy add-cluster-role-to-user cluster-admin system:serviceaccount:openshift-nmstate:nmstate-operator"
```

Comment:
I think we want to do this *after* we apply the nmstate manifests but before we are stuck in the verification loop.

Perhaps just run it after 3 failed validations of the `verify_remote_resource` command?